### DISCUSSION TITLE
integration with single cell genomic data

### DISCUSSION THREAD

**Discussion by soilmicrobiome** [metagenome-atlas/atlas#471] [Q&A]
Is it possible to run metagenomic samples and SAGs in the same run?  Would be nice to have one reconciled genecatalog/abundance table for all the samples (since in my case they are one data set).  But for the SAGs they don't need the full pipeline.  Is this possible or will the SAGs need to be run separately as "genomes"?  Thoughts on how to deal with SAG data, any suggestions?

**Comment by SilasK** [comment]
Good idea to include SAGs. However, I don't have experience with them? What would be a typical workflow for SAGs?

**Comment by soilmicrobiome** [comment]
I usually treat them like a genome.  So i run them with GTDB-Tk, predict genes, annotate and then i read map with the metagenome to get some sense of abundance.  But would be cool if they could some how be part of the pipeline just not sure how.

**Reply by SilasK** [reply]
Ok, so all the assembly is already done. So what would be needed is an option to import genomes, that's what you want. 

A little hacky version would be to add the genomes to the `genomes/genomes` folder. 

Maybe do 
```
cd atlas_working_dir
mv genomes old_genomes
mkdir  genomes

cp -r genomes genomes/genomes

```

And then you run atlas run genomes

**Reply by soilmicrobiome** [reply]
Not quite sure i follow. Would this be a way to integrate them into the MetaG sample i have (sample the SAGs were produced from)?  Or would i run the sags as there own sample set?

**Reply by SilasK** [reply]
I haven't worked with SAG samples. Could you explain me what you have done/ and want to do? 


If you want you can email me or ask for a videocall if this is easier. You can find the mail on my personal webpage.
